README.md

```diff
@@ -13,8 +13,8 @@ For Arch Linux users, it is available in AUR. [i915-sriov-dkms-git](https://aur.
 
 Tested kernel versions: 
 
-* `pve-kernel-6.1.0-1-pve`~`6.1.10-1-pve` on PVE VM Host
-* `gentoo-sources-6.1.0-gentoo`~`6.2.0-gentoo` on Gentoo VM Guest
+* `pve-kernel-6.2.9-1-pve` on PVE VM Host
```

Comment:
Is the 6.1.x PVE kernel no longer supported?

Reply:
Nope, it just means not tested on 6.1 PVE kernel.